src/core/messages.ts

```diff
@@ -261,7 +282,7 @@ export class DefaultMessages
 
     addListenerToChannelWithoutAttach({
       listener: this._processEvent.bind(this),
-      events: [MessageEvents.Created],
+      events: [RealtimeMessageTypes.ChatMessage, RealtimeMessageTypes.LegacyChatMessage],
```

Comment:
Bitten by our own CI :D We'll need to eslint exclude this.

Reply:
Removed as we have bumped the endpoints to use the new type now :)